modules/tables/src/lib/schema/field.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 Unfolded, Inc. All rights reserved.
+import assert from '../utils/assert';
+import {DataType} from '../types/arrow-like/type';
+
+// ArrowJS `Schema` API-compatible class for row-based tables (returned from `DataTable`)
```

Comment:
I see, so you don't depend on the Arrow JS library directly within the `tables` module. Makes a lot of sense; presumably it's easy to translate these fields into actual Arrow fields.

Reply:
Yes ArrowJS is still a bit too big a dependency for comfort, but my hope is eventually we can remove these helper classes. 

Until then we can trivially convert as you say.